fastlane/Fastfile

```diff
@@ -86,7 +86,7 @@ platform :ios do
       repo_name: repo_name,
       github_api_token: ENV["GITHUB_TOKEN"],
       changelog_latest_path: changelog_latest_path,
-      upload_assets: ['RevenueCat.framework.zip', 'RevenueCat.xcframework.zip']
```

Comment:
Not sure if a lot of people are using those but  just in case we should mention this in the changelog once we release this

Reply:
Yup: 
> remove `.framework` archives from releases

Also this is `breaking` so it'll be highlighted as a breaking change.